### PR TITLE
Minimum time to check for new IMs option in ACP is set to 1 by default

### PR DESCRIPTION
**Pull request by senky**
Minimum time to check for new IMs option in ACP is set to 1 by default, while it can be set to 2 at minimum:

```
The provided value for the setting “Minimum” is too low. The minimum acceptable value is 2.
```

This error is displayed even before you submit form.

This option should be set to `2` by default and checked in update script to be updated to `2` if it is set to `1`.

http://phpbbsocialnetwork.com/viewtopic.php?f=44&t=4290

### CODE REVIEW

File: git-tools/update_database_dev.php [hunk 1]
```diff
@@ -678,6 +684,37 @@ function show_usage()
...
+				FROM ' . SN_CONFIG_TABLE . '
+				WHERE config_name = "im_checkTime_min"';
+		$result = $db->sql_query($sql);
+		$im_checkTime_min = $db->sql_fetchfield('config_value');
+
+		if ($im_checkTime_min == 1)
+		{
+			$db->sql_query('UPDATE ' . SN_CONFIG_TABLE . ' SET config_value = "2" WHERE config_name = "im_checkTime_min"');
+
+			return 'Social Network::IM check time minimum was set to 2';
```
Comment (Kamahl19) [inline git-tools/update_database_dev.php:706]: I dont think we need to return a notice about this.

**Comment by Kamahl19** [comment]
MA but I think we should remove return messages

**Comment by senky** [comment]
Why? It is used in any other custom functions in UMIL. Admin should be noticed that something has been changed in his board without asking him before.

**Comment by Kamahl19** [comment]
allright

File: git-tools/update_database_dev.php (hunk 1 above)
Comment (Culprit-CZ) [inline git-tools/update_database_dev.php:706]: UMIL returns is necessary. We should give some info to output.

**Comment by senky** [comment]
d4ef4db merged this PR.

**Pull request closed without merging**